data_model/src/lib.rs

```diff
@@ -882,6 +883,64 @@ impl TryFrom<Value> for RegistrableBox {
     }
 }
 
+impl TryFrom<Value> for trigger::Trigger<FilterBox, Option<Vec<InstructionBox>>> {
```

Comment:
It seems like this `impl` is no longer needed and can be removed? (and one below)